kernel_module/legion-laptop.c

```diff
@@ -1712,6 +1712,12 @@ enum OtherMethodFeature {
 	OtherMethodFeature_C_U1 = 0x05010000,
 	OtherMethodFeature_TEMP_CPU = 0x05040000,
 	OtherMethodFeature_TEMP_GPU = 0x05050000,
+	OtherMethodFeature_FAN_FULLSPEED = 0x04020000,
```

Comment:
What is the source for that? Is there already a function that uses it that is not in this commit? Maybe you already have info how to use it. It is currently not used in this file.